src/app/config/urls.ts

```diff
@@ -72,8 +72,7 @@ export const LocationToPath: Record<Location, string> = {
   [Location.Blast]: '/blast',
   [Location.PeptideSearchResult]: '/peptide-search/:id/:subPage?',
   [Location.PeptideSearch]: '/peptide-search',
-  // TODO: check final URL for those
-  [Location.IDMappingResult]: '/id-mapping/:id/:subPage?',
+  [Location.IDMappingResult]: '/id-mapping/:id/:targetNS?',
```

Comment:
This probably needs to be inverted.
If you don't want an ID to be matched as a targetNS, explicitely state which are the supported namespaces (see below definition of `allSearchResultLocations`)

Reply:
I've put `targetNS` at the end because it can be empty (when the target is undefined).

Reply:
Still, since it's the opposite of the URL format given by the server I'm not too keen on going for a different pattern